internal/config/operations.go

```diff
@@ -0,0 +1,38 @@
+// Copyright 2020 Layer5, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package config
+
+import (
+	"github.com/layer5io/meshery-adapter-library/adapter"
+	"github.com/layer5io/meshery-adapter-library/meshes"
+)
+
+var (
+	ServiceName  = "service_name"
+	OSMOperation = "osm"
+)
+
+func getOperations(op adapter.Operations) adapter.Operations {
+	op[OSMOperation] = &adapter.Operation{
+		Type:        int32(meshes.OpCategory_INSTALL),
+		Description: "OSM Service Mesh",
+		Versions:    []adapter.Version{"v0.3.0", "v0.2.0"},
+		Templates:   adapter.NoneTemplate,
+		AdditionalProperties: map[string]string{
+			ServiceName: OSMOperation,
```

Comment:
@kumarabd should this be one in Services field?

Reply:
@kumarabd still not  quite sure what we mean by 'Service' here. It's not used in the code, as far as I can see.
My understanding is/was that this is the name of the service used to retrieve endpoint information for?

Reply:
@mgfeller you're right. Its not being used anywhere, I just removed it.